action.yml

```
@@ -38,6 +38,10 @@ inputs:
     description: "The arugments to use when pushing the repository"
     default: '--tags --force --prune'
     required: false
+  GITHUB_REF:
+    description: "The branch name to push to when not pushing all refs"
+    default: ${{ github.ref }}
```

```suggestion
    default: ${{ github.head_ref || github.ref_name }}
```

Perhaps this is best so that I don't have to clutter the documentation?